Sources/Testing/Running/Configuration.swift

```diff
@@ -128,17 +128,38 @@ public struct Configuration: Sendable {
   ///
   /// By default, all tests are run and no filter is set.
   public mutating func setTestFilter(toMatch selection: Set<Test.ID>?) {
-    setTestFilter(toMatch: selection.map(Test.ID.Selection.init))
+    setTestFilter(toMatch: selection, includeHiddenTests: false)
   }
-  
+
+  /// The granularity to enforce test filtering.
```

Comment:
This brief doesn't seem to make sense in context? I realize you copied it from the existing method—we should probably fix it. :)

Also, and I know this sounds like nitpicking: we need a blank line after the brief before `- Parameters:` to ensure DocC correctly parses this comment.